server/fishtest/util.py

```diff
@@ -296,6 +297,7 @@ def format_results(run_results, run):
     return result
 
 
+@lru_cache # A single hash lookup is much cheaper than re-parsing a string every time
```

Comment:
We have only few TCs, so we can use safely the slightly faster `lru_cache(maxsize=None)` or the alias `cache` introduced with pyhton 3.9
https://docs.python.org/3/library/functools.html#functools.cache